code/modules/clothing/gloves/color.dm

```diff
@@ -65,6 +65,7 @@
 	desc = "Rudimentary gloves that aid in carrying."
 	icon_state = "goligloves"
 	item_state = "goligloves"
+	can_be_cut = FALSE //no turning goliath gloves into fingerless gloves
```

Comment:
please don't have this it'll cause cornflakes with the BIG bow pr

Reply:
fine, but shamans spawn with the clothing set that has those, and them being cuttable into fingerless gloves seems weird